app/views/content/service_privacy_notice.md

```diff
@@ -112,16 +112,17 @@ Processing references includes:
 
 ### Building a better teacher training application process
 
-We use some user data to improve our services. For example, we’ll look at any feedback you give us about our services so we can improve them. If you have given us your consent we may also contact you to participate in user research.
+We use some user data to improve our services. For example, we’ll look at any feedback you give us about our services so we can improve them.
 
 #### If you’re a referee
 
-We ask if you’re willing to participate in user research when you submit a reference.
+We ask if you’re willing to participate in user research when you submit a reference. If you have given us your consent we may contact you to participate in user research.
 If you want to withdraw your consent to participate in user research at any time, you can contact us at becomingateacher@digital.education.gov.uk.
 
 #### If you’re a candidate
 
-We ask for your consent to contact you to participate in user research when you submit your application. This helps us improve our services and policies and further our purpose of recruiting high quality teachers. If you want to withdraw your consent to participate in user research at any time, you can contact us at becomingateacher@digital.education.gov.uk.
+We may contact you to ask if you would like to participate in user research. This helps us improve our services and policies and further our purpose of recruiting high quality teachers. You do not have to participate, and choosing to participate or not won’t affect your application in any way. You can opt out of being contacted for user research at any time using the form below, you will still receive all other communications about your application.
+[Opt out of being contacted for research](https://forms.gle/dkZtK1g8ipKTD3s66).
```

Comment:
I wonder slightly if "opt out" is jargon and if "ask not to be contacted" would be plainer English? That said, maybe it's common enough to be well known now? @johndoates